apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -131,8 +133,6 @@ export const AIAssistant = ({ className }: AIAssistantProps) => {
   const currentSchema = searchParams?.get('schema') ?? 'public'
   const currentChat = snap.activeChat?.name
 
-  const { ref } = useParams()
-  const org = useSelectedOrganization()
```

Comment:
looked like these were duplicated calls to `useParams()` and `useSelectedOrganization()`